py-glaredb/tests/test_logical_plan.py

```diff
@@ -1,6 +1,11 @@
 import pytest
 import glaredb
 
+def pydict_sort(pd):
+    res = dict()
+    for key in sorted(pd):
+        res[key] = sorted(pd[key])
+    return res
```

Comment:
This logic isn't what we're looking for here. We don't want to sort columns independently.

For example, this test case would fail every time with this logic:

```
def test_compose2():
    con = glaredb.connect()
    con.execute("create table tbl (a int, b int);")
    con.execute("insert into tbl values (1, 2);")
    con.execute("insert into tbl values (3, 4);")
    con.execute("insert into tbl values (5, 2);")
    intermediate = con.sql("select * from tbl where a > 2;")
    out_1 = pydict_sort(intermediate.to_arrow().to_pydict())
    expected = {"a": [3, 5], "b": [4, 2]}
    assert out_1 == expected
```

I would just add an `order by a` to the intermediate query.